test/docker/mac-os-test-env/start.sh

```diff
@@ -10,4 +10,7 @@ echo "*** Run 'debug' or 'debug <path-to-test-file>' to get into debug mode."
 echo "*** In Chrome open chrome://inspect to access the debugging tools."
 echo
 
+npm install
+npm install -g mocha
+
 /bin/bash
```

Comment:
missing newline at end of file. i wouldn't have noticed but github showed me. what editor do you use? i've configured vscode to always strip trailing whitespace and add a final newline on save.